### ISSUE TITLE
Feature request

### ISSUE THREAD

**Issue by hellishvictor** [omeriko9/LSO2MIDI#1]
Hi, good work creating this one! Would be awesome if there were a MIDI2LSO for quick convertion (to Logic Platinum 5).
Cheers.

**Comment by omeriko9** [comment]
Interesting idea, I'll look into it on my weekend :)

**Comment by omeriko9** [comment]
Wait, just remembered that Logic Platinum 5 has a "Import MIDI" option, why isn't it good enough for your needs?

**Comment by hellishvictor** [comment]
Yeah, but it would be faster using a batch file with a "for" loop and the command line MIDI2LSO; I have hundreds of midi patters that I'll like to check on Logic mapped. If where possible specify a parameter to load them to a predefined LSO used as template or just directly the midi tracks to AudioInst would be perfect.

**Comment by hellishvictor** [comment]
Yeah, but it would be faster using a batch file with a "for" loop and the command line MIDI2LSO; I have hundreds of midi patters that I'll like to check on Logic mapped. If where possible specify a parameter to load them to a predefined LSO used as template, or just load directly the tracks on the midi file to AudioInst, would be perfect.
